lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -753,4 +727,122 @@ sub _create_download_lists {
     }
 }
 
+sub _schedule_from_yaml ($self, $args, $skip_chained_deps, @load_yaml_args) {
+    my $data = eval { load_yaml(@load_yaml_args) };
+    if (my $error = $@) { return {error_message => "Unable to load YAML: $error"} }
```

Comment:
This doesn't seem to be caught in the tests?

Reply:
It is covered. It is the check for `Could not open 'does-not-exist.yaml' for reading: No such file or directory`.

Reply:
I added this error-message-prefix now to the checks (in both cases).